README.md

```diff
@@ -11,7 +11,7 @@ Lc0 can be acquired either via a git clone or an archive download from GitHub. B
 
 For essentially all purposes, including selfplay game generation and match play, we highly recommend using the latest `release/version` branch (for example `release/0.25`), which is equivalent to using the latest version tag.
 
-Versioning follows the Semantic Versioning guidelines, with major, minor and patch sections. The training server enforces game quality using the versions output by the client and engine.
+Versioning follows the Semantic Versioning guidelines, with major, minor and patch sections. The training server enforces game quality using the version's output by the client and engine.
```

Comment:
This correction looks wrong to me; it is not the version outputting things, but the versions which are output by client and engine. If anything sounds wrong grammatically, it would be `output` being used instead of the alternative `outputted`, see https://conjugator.reverso.net/conjugation-english-verb-output.html

Reply:
I think it would be best to rewrite the sentence. Something like:

"The training server enforces game quality by checking the version number of the client and engine."